Content for speculative loading (#29183)
mdn/content: files/en-us/web/api/document/index.md

  - : Returns an {{DOMxRef("HTMLCollection")}} of the available plugins.
- {{DOMxRef("Document.pointerLockElement")}} {{ReadOnlyInline}}
  - : Returns the element set as the target for mouse events while the pointer is locked. `null` if lock is pending, pointer is unlocked, or if the target is in another document.
- {{DOMxRef("Document.prerendering")}} {{ReadOnlyInline}} {{experimental_inline}}

>   - : Returns a boolean — `true` if the document is currently in the process of prerendering, as initiated via the [Speculation Rules API](/en-US/docs/Web/API/Speculation_Rules_API), and `false` if not.

Reviewer: Nothing wrong, but perhaps
```suggestion
  - : Indicates whether the document is currently in the process of prerendering, as initiated via the [Speculation Rules API](/en-US/docs/Web/API/Speculation_Rules_API).
```

Author: Fair call, and yours is more in keeping with the style on this page, but I like the idea of being able to get what you need from this page and not having to click through to the subpage if you don't need to.

Saying that, I think mine was a bit wordy, so I've updated the wording to a compromise:

> Returns a boolean that indicates whether the document is currently in the process of prerendering, as initiated via the [Speculation Rules API](/en-US/docs/Web/API/Speculation_Rules_API).

From this you can definitely infer what you need to know, but it is more concise.
